package.json

```diff
@@ -7,19 +7,32 @@
     "author": "CXL <leho@cxl.com>",
     "license": "MIT",
     "peerDependencies": {
-        "eslint": "^7.7.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "eslint": "^6.8.0",
+        "husky": "^4.3.6",
+        "lint-staged": "^10.5.3",
+        "prettier": "npm:wp-prettier@2.0.5"
     },
     "scripts": {
         "postinstall": "node postinstall.js"
     },
     "devDependencies": {
-        "eslint": "^7.7.0",
-        "eslint-config-prettier": "^6.11.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "@10up/eslint-config": "^2.1.1",
+        "@10up/stylelint-config": "^1.0.11",
+        "@wordpress/eslint-plugin": "^7.4.0",
+        "babel-eslint": "^10.1.0",
+        "eslint": "^6.8.0",
+        "eslint-config-airbnb": "^18.1.0",
+        "eslint-config-airbnb-base": "^14.1.0",
+        "eslint-config-prettier": "^6.10.0",
+        "eslint-plugin-import": "^2.20.1",
+        "eslint-plugin-jsdoc": "^22.0.1",
+        "eslint-plugin-jsx-a11y": "^6.2.3",
+        "eslint-plugin-prettier": "^3.1.2",
+        "eslint-plugin-react": "^7.19.0",
+        "eslint-plugin-react-hooks": "^2.5.0",
+        "husky": "^4.3.6",
+        "lint-staged": "^10.5.3",
+        "prettier": "npm:wp-prettier@2.0.5",
```

Comment:
Things are both in dev and peer?

Reply:
I saw you had them in both, so I tried to do the same.
Will remove from peer deps.

Reply:
> I saw you had them in both, so I tried to do the same.
> Will remove from peer deps.

Oh, I forgot. Maybe that's actually the right technique?

Reply:
> > I saw you had them in both, so I tried to do the same.
> > Will remove from peer deps.
> 
> Oh, I forgot. Maybe that's actually the right technique?

Will test it out... by removing peer deps and see how it works...

Reply:
Removed the peer dependency requirement, and seems to work nicely so far.

Reply:
Should've researched better. peerDepedencies is the official recommendation https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

Reply:
True as per above suggestion.

But we are instead including this 
https://github.com/conversionxl/lint-staged-config
into projects, and extending it?

See these examples:

https://github.com/conversionxl/jobs-theme/blob/cc4a540b5da3a5b7111d4d41c7459dfecedca0f8/lint-staged.config.js
https://github.com/conversionxl/jobs-theme/blob/cc4a540b5da3a5b7111d4d41c7459dfecedca0f8/husky.config.js
https://github.com/conversionxl/jobs-theme/blob/cc4a540b5da3a5b7111d4d41c7459dfecedca0f8/.stylelintrc

etc...

So these packages are already available in projects where its being utilized...

Or should include packages **peerDepedencies** ?

Reply:
> Or should include packages peerDepedencies ?

Yes, please.

Reply:
Check now...